src/lib.rs

```diff
@@ -255,6 +279,25 @@ pub fn encode_config_buf(input: &[u8], config: Config, buf: &mut String) {
             raw.push(0x3d);
         }
     }
+
+    if config.line_size.is_some() && config.line_size.unwrap() > 0 {
```

Comment:
This could use `config.line_size.unwrap_or(0) > 0`, or even store that in a local to avoid the unwrap below